checkPropTypes.js

```diff
@@ -54,7 +56,7 @@ function checkPropTypes(typeSpecs, values, location, componentName, getStack) {
             var err = Error(
               (componentName || 'React class') + ': ' + location + ' type `' + typeSpecName + '` is invalid; ' +
               'it must be a function, usually from the `prop-types` package, but received `' + typeof typeSpecs[typeSpecName] + '`.' +
-              'This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.'
```

Comment:
i think it's much better to leave the strings inline; gzip compresses them and dead code elimination is more able to remove them.

Reply:
> i think it's much better to leave the strings inline; gzip compresses them and dead code elimination is more able to remove them.

Can you provide more information about why inline strings are better for gzip compression and dead code elimination? Thanks in advance.